release/models/bgp/openconfig-bgp-common-multiprotocol.yang

```diff
@@ -24,7 +24,13 @@ submodule openconfig-bgp-common-multiprotocol {
     for multiple protocols in BGP. The groupings are common across
     multiple contexts.";
 
-  oc-ext:openconfig-version "9.4.0";
+  oc-ext:openconfig-version "9.4.1";
+
+  revision "2023-06-27" {
+    description
+      "Clarify bgp remote-port description";
+    reference "9.4.1";
+  }
```

Comment:
Should we update the following as well w/ similar explanation for consistency?

/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/transport/state/
local-port
description:
Local TCP port being used for the TCP session supporting the BGP session